wazuh/wazuh-kibana-app
# Added filters to Vulnerabilities - Inventory flyout details

Hi team,

this PR adds filters to `Vulnerabilities -> Inventory` Flyout.

Closes #3739 

How to test:

- Go to `Vulnerabilities -> Inventory`
- Select an agent with vulnerabilities
- Select a record to open the flyout
- Filter a field from Details section
- Test the `Export Formatted` button from the table (_the file should have only the filtered records_)

![Peek 2021-12-21 15-02](https://user-images.githubusercontent.com/9343732/146943028-b4148062-7cad-4fb2-93cf-322786176c65.gif)
